src/models/User.model.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  Model,
+  Table,
+  Column,
+  Default,
+  PrimaryKey,
+  Unique,
+  BelongsTo,
+  UpdatedAt,
+  CreatedAt,
+  IsUUID,
+  Length,
+  BeforeCreate,
+  BeforeUpdate,
+  ForeignKey,
+  IsEmail,
+} from "sequelize-typescript";
+import { v4 } from "uuid";
+import { RoleModel } from "./Role.model";
+import { DATE, STRING, UUID, UUIDV4 } from "sequelize";
```

Comment:
First of all import package.json dependencies, than add blank line and import internal project files.